Build/PackageFiles/Dotnet_precompiled/ItemTemplates_v3.x.nuspec

```diff
@@ -25,17 +25,18 @@
     <file src="..\..\..\Functions.Templates\Templates\DurableFunctionsEntityFunction-CSharp-2.x\**" target="content\DurableFunctionsEntityFunction-CSharp" exclude="**\*.csx;**\*.md;**\*.dat;**\function.json;**\metadata.json" />
     <file src="..\..\..\Functions.Templates\Templates\DurableFunctionsEntityClass-CSharp-2.x\**" target="content\DurableFunctionsEntityClass-CSharp" exclude="**\*.csx;**\*.md;**\*.dat;**\function.json;**\metadata.json" />
-    <file src="..\..\..\Functions.Templates\Templates\SendGrid-CSharp-4.x\**" target="content\SendGrid-CSharp" exclude="**\*.csx;**\*.md;**\*.dat;**\function.json;**\metadata.json" />
+    <file src="..\..\..\Functions.Templates\Templates\SendGrid-CSharp-5.x\**" target="content\SendGrid-CSharp" exclude="**\*.csx;**\*.md;**\*.dat;**\function.json;**\metadata.json" />
     <file src="..\..\..\Functions.Templates\Templates\EventHubTrigger-CSharp-5.x\**" target="content\EventHubTrigger-CSharp" exclude="**\*.csx;**\*.md;**\*.dat;**\function.json;**\metadata.json" />
-    <file src="..\..\..\Functions.Templates\Templates\ServiceBusQueueTrigger-CSharp\**" target="content\ServiceBusQueueTrigger-CSharp" exclude="**\*.csx;**\*.md;**\*.dat;**\function.json;**\metadata.json" />
-    <file src="..\..\..\Functions.Templates\Templates\ServiceBusTopicTrigger-CSharp\**" target="content\ServiceBusTopicTrigger-CSharp" exclude="**\*.csx;**\*.md;**\*.dat;**\function.json;**\metadata.json" />
+    <file src="..\..\..\Functions.Templates\Templates\ServiceBusQueueTrigger-CSharp-5.x\**" target="content\ServiceBusQueueTrigger-CSharp" exclude="**\*.csx;**\*.md;**\*.dat;**\function.json;**\metadata.json" />
+    <file src="..\..\..\Functions.Templates\Templates\ServiceBusTopicTrigger-CSharp-5.x\**" target="content\ServiceBusTopicTrigger-CSharp" exclude="**\*.csx;**\*.md;**\*.dat;**\function.json;**\metadata.json" />
     <file src="..\..\..\Functions.Templates\Templates\EventGridTrigger-CSharp-3.x\**" target="content\EventGridTrigger-CSharp" exclude="**\*.csx;**\*.md;**\*.dat;**\function.json;**\metadata.json" />
+    <file src="..\..\..\Functions.Templates\Templates\EventGridTrigger-FSharp-3.x\**" target="content\EventGridTrigger-FSharp" exclude="**\*.csx;**\*.md;**\*.dat;**\function.json;**\metadata.json" />
     <file src="..\..\..\Functions.Templates\Templates\IoTHubTrigger-CSharp\**" target="content\IotHubTrigger-CSharpVS" exclude="**\*.csx;**\*.md;**\*.dat;**\function.json;**\metadata.json" />
-    <file src="..\..\..\Functions.Templates\Templates\CosmosDBTrigger-CSharp\**" target="content\CosmosDBTrigger-CSharp" exclude="**\*.csx;**\*.md;**\*.dat;**\function.json;**\metadata.json" />
+    <file src="..\..\..\Functions.Templates\Templates\CosmosDBTrigger-CSharp-5.x\**" target="content\CosmosDBTrigger-CSharp" exclude="**\*.csx;**\*.md;**\*.dat;**\function.json;**\metadata.json" />

```

Comment:
Should this be 4.x to align with the current version of the Cosmos extension?

Reply:
The value here needs to match the folder name where the templates reside. See my comment above for why I named the folder `-5.x`, but I could easily rename it to `-4.x`